pkg/sdn/plugin/multitenant.go

```diff
@@ -111,6 +111,11 @@ func (mp *multiTenantPlugin) UpdateNetNamespace(netns *osapi.NetNamespace, oldNe
 }
 
 func (mp *multiTenantPlugin) DeleteNetNamespace(netns *osapi.NetNamespace) {
+	// Force multicast disabled for this VNID.  It hasn't been deleted from
+	// the VNIDMap until after this function returns, but UpdateLocalMulticastRules()
```

Comment:
It looks like nothing depends on that behavior though, so maybe it would be simpler to just change that.

Also, it looks like networkpolicy.go is broken too. (Even more so; it doesn't even make a *failed* attempt to clean up.)